internal/backend/mqtt/auth/azure_iot_hub.go

```diff
@@ -0,0 +1,108 @@
+package auth
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"crypto/tls"
+	b64 "encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/url"
+	"strconv"
+	"time"
+
+	mqtt "github.com/eclipse/paho.mqtt.golang"
```

Comment:
`mqtt` redundant alias, consider removing it.

Reply:
Got it!